guides/advanced_guides/tokenization.md

```diff
@@ -0,0 +1,31 @@
+# Tokenization
+
+**Tokenization** is the act of taking a sentence or phrase and splitting it into smaller units of language, called tokens. It is the first step of document indexing in the Meilisearch engine, and is a critical factor in the quality of search results.
+
+Breaking sentences into smaller chunks requires understanding where one word ends and another begins, making tokenization a highly complex and language-dependant task. MeiliSearch's solution to this problem is a **modular tokenizer** that follows different processes, called **pipelines**, based on the type of character it detects. This allows it to function on datasets from multiple languages with minimal setup.
+
+Currently, the MeiliSearch tokenizer has full support for four languages: English, Chinese, Japanese, and Korean. It can also effectively tokenize any language that uses the Latin alphabet.
```

Comment:
```suggestion
Currently, the MeiliSearch tokenizer has full support for four languages: 
- English  🇬🇧 (and all Latin based languages)
- Chinese 🇨🇳
- Japanese 🇯🇵
- Korean 🇰🇷
```
Maybe an emphasis on the supported language? 
This should be added as well in the readme.md of the core meilisearch repo: https://github.com/meilisearch/meilisearch (do you want me to make an issue about it?)

Reply:
Great change. Yes, please :)

Reply:
Since you are the creator of the PR, you should make the commits. If you like my suggestion you can apply it by clicking here: 
<img width="861" alt="Screenshot 2021-01-11 at 21 30 07" src="https://user-images.githubusercontent.com/33010418/104234770-3817c500-5454-11eb-9463-5fb6d21962c2.png">